stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -290,7 +294,21 @@ public DefaultInspectorModulesBuilder runtimeRepl(RuntimeReplFactory factory) {
      * {@link Context#getDatabasePath} method will be used by default if not overridden here.
      */
     public DefaultInspectorModulesBuilder databaseFiles(DatabaseFilesProvider provider) {
-      mDatabaseFiles = provider;
+      mDatabaseFilesProvider = provider;
+      return this;
+    }
+
+    /**
+     * Extend and provide additional database drivers. Currently two database drivers are supported
+     * in this lib: <br>
+     *   1. <code>SqliteDatabaseDriver</code> - Presents sqlite databases and all tables of the app.<br>
```

Comment:
Yep. Some time ago when I used Eclipse, java docs were really useful and presented nicely and formatted. Then I used correct tags. In Android Studio, it's just not the same. But, ye, for java docs generation it's important I think. Will use the right tags in the future pull requests ;) Thanks.